stl/inc/experimental/generator

```diff
@@ -13,12 +13,22 @@
 #endif
 #include <memory>
 
-#if defined(__cpp_impl_coroutine)
+#ifdef __clang__
+_EMIT_STL_ERROR(STL1009, "The <experimental/coroutine>, <experimental/generator>, and <experimental/resumable> "
+                         "headers do not support Clang, but the C++20 <coroutine> and C++23 <generator> headers do.");
+#elif !defined(_SILENCE_EXPERIMENTAL_COROUTINE_DEPRECATION_WARNINGS)
+_EMIT_STL_ERROR(STL1011, "<experimental/generator> is deprecated by Microsoft and will be REMOVED IN THE NEAR FUTURE. "
+                         "It is superseded by the C++23 <generator> header. You can define "
+                         "_SILENCE_EXPERIMENTAL_COROUTINE_DEPRECATION_WARNINGS to suppress this error for now.");

```

Comment:
The same macro is used to suppress two different messages. They are different on which header is deprecated.

Reply:
I felt it was simpler to have a single silencing macro, although I suppose I could unify the messages.

Reply:
Unified, thanks.